public/core.d.ts

```diff
@@ -1212,8 +1212,9 @@ declare namespace JMap {
        * JMap.Layer.DynamicFilter.set([{
        *   layerId: 3,
        *   conditions: [{
+       *     layerId: 3,
```

Comment:
As-tu testé l'exemple? Il me semble qu'il manque des params obligatoire dans les conditions

Reply:
Oui, ça fonctionnait. Tu pensais à quel params ?

Reply:
quand je regarde l'interface JDynamicFilterCondition, je vois un id qui n'est pas optionnel, mais on en crée possiblement un on-the-fly si tu n'as pas rencontré d'erreur, faudrait valider et ajuster la doc au besoin